modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/css/main.scss

```diff
@@ -162,20 +162,51 @@
 		}
 	}
 
+	h3.component-title {
+		font-size: $h3-font-size;
+
+		@include media-breakpoint-down(sm) {
+			font-size: $h3-font-size-mobile;
+		}
+	}
+
+	.message-user-display {
+		margin-bottom: 0.5rem;
+		margin-top: 0.5rem;
+	}
+
+	.social-interaction {
+		display: flex;
+		margin-left: -0.5rem;
+
+		> div {
+			margin-right: 0.5rem;
+		}
+	}
+
 	@include media-breakpoint-up(sm) {
 		.message-user-display {
-			width: 50%;
+			width: calc(100% - 198px);
```

Comment:
It looks too specific for this situation, probably we need a more standard solution to apply it to lima widgets

Reply:
`message-user-display` is scoped to:

```
.portlet-asset-publisher,
.portlet-message-boards,
.portlet-trash {}
```

The only other place what uses the class `message-user-display` is the `akismet` module. I think this one is pretty safe unless they want to reimplement the placement of thumbs up, thumbs down, flag buttons.